turtlestar-evaluationtest.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from pathlib import Path
+from shutil import copyfile
+from tempfile import TemporaryDirectory
+
+from rdflib.exceptions import ParserError
+
+from rdflib import Graph
+from rdflib.util import guess_format
+
+
+from rdflib.plugin import register
+from rdflib.parser import Parser
+from rdflib.serializer import Serializer
+
+import rdflib
+from rdflib import URIRef
+from rdflib.namespace import RDF
+from rdflib.namespace import FOAF
+
+g = Graph()
+g.parse(data="test/turtlestar-evaluation/turtle-star-eval-01.ttl", format = "ttls")
+print(g.serialize(format = "ttlstar"))
+
+g = Graph()
+g.parse("test/turtlestar-evaluation/turtle-star-eval-02.ttl", format = "ttls")
+print(g.serialize(format = "ttlstar"))
```

Comment:
It would be best to attempt some manner of equivalence checking here. Blank nodes may complicate matters, but we have [`GraphHelper.assert_sets_equals`](https://github.com/RDFLib/rdflib/blob/3a418218d6bcdb46f78342e14c024063e2f53e71/test/utils/__init__.py#L289) which could be adapted to deal with RDFStar nodes, the best would be to use `bnode_handling = BNodeHandling.COLLAPSE`.

We should actually change `GraphHelper.assert_sets_equals` to compare size also now that I think about it.